python/ray/ml/BUILD

```diff
@@ -36,6 +36,14 @@ py_test(
     deps = [":ml_lib"]
 )
 
+py_test(
+    name = "test_xgboost_lightgbm_predictors.py",
+    size = "small",
+    srcs = ["tests/test_xgboost_lightgbm_predictors.py.py"],
+    tags = ["team:ml", "exclusive"],
+    deps = [":ml_lib"]
+)
+
```

Comment:
can we split this into xgboost and lightgbm separately? The script can remain the same, but maybe with different flags